docs/src/examples/gol_lazar_belta.jl

```diff
@@ -0,0 +1,164 @@
+#md #
+# The goal is to take the state vector toward a target set **XT** by visiting one of the squares
+# **A** or **B** and avoiding the obstacles **O1** and **O2**
+
+
+# First, let us import [CDDLib](https://github.com/JuliaPolyhedra/CDDLib.jl/tree/v0.6.3), 
+# [GLPK](https://github.com/jump-dev/GLPK.jl), [OSQP](https://github.com/oxfordcontrol/OSQP.jl),
+# [JuMP](https://github.com/jump-dev/JuMP.jl), [Pavito](https://github.com/jump-dev/Pavito.jl)
+# and [Ipopt](https://github.com/jump-dev/Ipopt.jl)
+import CDDLib
+import GLPK
+import OSQP
+using JuMP
+import Pavito
+import Cbc
+import Ipopt
+
+# At this point we import Dionysos
+using Dionysos
+
+# And the file defining the hybrid system for this problem
+include(joinpath(dirname(dirname(pathof(Dionysos))), "examples", "gol_lazar_belta.jl"))
+
+# Now we instantiate our system using the function provided by [gol_lazar_belta.jl](@__REPO_ROOT_URL__/examples/gol_lazar_belta.jl)
+system = gol_lazar_belta(CDDLib.Library());
+
+
+# Then, we define initial conditions (continuous and discrete states) to this system
+# and set **N** as the search depth, i.e., the number of allowed time steps.
+
+x0 = [1.0, -6.0];
+q0 = 3;
+
+N = 11;
+
+# We instantiate our Optimal Control Problem by defining the state and transition costs
+state_cost = Fill(ZeroFunction(), nmodes(system))
+transition_cost = QuadraticControlFunction(ones(1, 1))
+
+problem = OptimalControlProblem(
+    system,
+    q0, x0,
+    Fill(state_cost, N),
+    Fill(Fill(transition_cost, ntransitions(system)), N),
+    system.ext[:q_T],
+    N
+);
+
+
+
+# Finally, we select the method presented in [[2]](#2) as our optimizer
+
+qp_solver = optimizer_with_attributes(
+    OSQP.Optimizer,
+    "eps_abs" => 1e-8,
+    "eps_rel" => 1e-8,
+    "max_iter" => 100000,
+    MOI.Silent() => true
+);
+
+mip_solver = optimizer_with_attributes(
+    Cbc.Optimizer,
+    MOI.Silent() => true
+);
+
+cont_solver = optimizer_with_attributes(
+    Ipopt.Optimizer,
+    MOI.Silent() => true
+);
+
+miqp_solver = optimizer_with_attributes(
+    Pavito.Optimizer,
+    "mip_solver" => mip_solver,
+    "cont_solver" => cont_solver,
+    MOI.Silent() => true
+);
+
+
+algo = optimizer_with_attributes(BemporadMorari.Optimizer, 
+    "continuous_solver" => qp_solver, 
+    "mixed_integer_solver" => miqp_solver,
+    "indicator" => false, 
+    "log_level" => 0
+);
+
+# and use it to solve the given problem, with the help of the abstraction layer 
+# MathOptInterface provided by [JuMP](https://github.com/jump-dev/JuMP.jl)  
+optimizer = MOI.instantiate(algo)
+MOI.set(optimizer, MOI.RawParameter("problem"), problem)
+@time MOI.optimize!(optimizer)
```

Comment:
Probably better to show `MOI.get(optimizer, MOI.SolveTime())` otherwise, you'll show compile time mostly.

Reply:
Done